### ISSUE TITLE
Innclude batch size in loss function

### ISSUE THREAD

**Issue by pushkarkadam** [pushkarkadam/handpose#32]
- [x] Unit test
- [x] Run a test run on dataset

**Comment by pushkarkadam** [comment]
https://stackoverflow.com/questions/61092523/what-is-running-loss-in-pytorch-and-how-is-it-calculated
